lib/streams/documentStream.js

```diff
@@ -28,7 +28,22 @@ function createDocumentStream(id_prefix, stats) {
         if (record.POSTCODE) {
           addrDoc.setAddress( 'zip', record.POSTCODE );
         }
-
+        
+        if (useOaAdminFields) {
+          if (record.CITY) {
+            addrDoc.parent.locality.push(record.CITY);
+          }
+
+          if (record.DISTRICT) {
+            addrDoc.parent.county.push(record.DISTRICT);
```

Comment:
Hmm, looking through OA, their district field is unfortunately a little inconsistent. In the US, it's usually county, but in other place it looks like it's sometimes a city neighbourhood. It's probably ok, and perhaps we can find the exceptions to the rule and program them in somehow.